lib/logger.js

```diff
@@ -39,6 +39,11 @@ x.test = function (props) {
 		return;
 	}
 
+	if (props.skip) {
+		log.write('  ' + chalk.cyan('- ' + props.title));
```

Comment:
Is cyan really the best color for skipped? Just asking. It should at least be `.dim` too.